.gitignore

```diff
@@ -11,3 +11,4 @@
 /public/system
 /tags
 /tmp/*
+todo.md
```

Comment:
Perhaps have this in your global .gitconfig? Not all of us use that and we try to keep the settings project vs developer specific.